x-pack/legacy/plugins/siem/public/pages/detection_engine/detection_engine_no_signal_index.tsx

```diff
@@ -14,7 +14,7 @@ export const DetectionEngineNoIndex = React.memo(() => (
   <EmptyPage
     actionPrimaryIcon="documents"
     actionPrimaryLabel={i18n.GO_TO_DOCUMENTATION}
-    actionPrimaryUrl={documentationLinks.siem}
+    actionPrimaryUrl={`${documentationLinks.siem}`}
```

Comment:
documentationLinks.siem seems to be an object, would we want to convert it to string here?